stylesheets/application.css

```diff
@@ -1073,6 +1073,11 @@ a[class*="icon-"]:hover {
   background-position-x: 4px;
 }
 
+#content .contextual .drdn-items a {
+  display: block;
+  width: 100%;
```

Comment:
`display: block` solves the issue, so `width: 100%` may be unnecessary. When `width: 100%` is applied, the button area appears to overflow, as shown in the screenshots below.

![CleanShot 2025-05-16 at 19 45 21](https://github.com/user-attachments/assets/d1bf0107-1a8c-468e-a792-a4b3d0c1c61c)

With `width:100%` removed:
![CleanShot 2025-05-16 at 19 46 09](https://github.com/user-attachments/assets/59a9d8e7-d97e-47ea-acfd-e33270a0f99e)

Reply:
<img width="530" alt="screenshot 2025-05-19 9 19 23（3）" src="https://github.com/user-attachments/assets/41aa199f-715d-40a9-ac1f-87ee414d0814" />

Indeed, the width was excessively large due to the width: 100%. I resolved the issue by removing the width: 100% line.
https://github.com/redmica/redmine_theme_kodomo_midori/pull/6/commits/b498d34b189ea1b7c3637d141de9936fc92643ea